download/index.md

```diff
@@ -45,7 +45,7 @@ cssid: download-pricing
 <tr><td>License</td><td><a href="/licenses/flyway-community">Apache v2</a></td><td><a href="/licenses/flyway-teams">Commercial</a></td></tr>
 <tr><td>Maven Repository</td><td><i class="fa fa-check"></i></td><td><i class="fa fa-check"></i></td></tr>
 <tr><td>Source Code included</td><td><i class="fa fa-check"></i></td><td><i class="fa fa-check"></i></td></tr>
-<tr><td>Support</td><td>Community</td><td>Email (2 business days)</td></tr>
+<tr><td>Support</td><td><a href="https://github.com/flyway/flyway/issues">Community</a></td><td>Email (2 business days)</td></tr>
```

Comment:
I feel like the link here should go to https://flywaydb.org/community/ because I think support here can be interpreted as both "how do I raise issues" and "how can I get help with doing x"

Reply:
done in latest commit